lib/livebook_web/live/hub/secret_list_component.ex

```diff
@@ -19,66 +19,44 @@ defmodule LivebookWeb.Hub.SecretListComponent do
           No secrets here... yet!
         </.no_entries>
         <div :if={@secrets != []}>
-          <table class="min-w-full divide-y divide-gray-300">
-            <thead>
-              <tr>
-                <th
-                  scope="col"
-                  class="py-3.5 pl-4 pr-3 text-left text-sm font-semibold text-gray-900 sm:pl-6"
+          <.table id="hub-secrets-table" rows={@secrets} show_action_label={true}>
+            <:col :let={secret} label="Name"><%= secret.name %></:col>
+            <:action :let={secret}>
+              <span class="tooltip left" data-tooltip="Edit">
+                <.link
+                  id={"hub-secret-#{secret.name}-edit"}
+                  patch={"/#{@edit_path}/#{secret.name}"}
+                  type="button"
+                  role="menuitem"
+                  class="icon-button"
                 >
-                  Name
-                </th>
-                <th
-                  scope="col"
-                  class="py-3.5 pl-3 pr-6 text-right text-sm font-semibold text-gray-900 sm:pr-8"
+                  <.remix_icon icon="edit-fill" class="text-lg" />
+                </.link>
+              </span>
+              <span class="tooltip left" data-tooltip="Delete">
```

Comment:
You are meant to put each button inside an action:

```suggestion
              </span>
            </:action>
            <:action :let={secret}>
              <span class="tooltip left" data-tooltip="Delete">
```